phpunit.xml.dist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<phpunit
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+        colors="true"
+        verbose="true"
+        failOnRisky="true"
+        beStrictAboutChangesToGlobalState="true"
+        beStrictAboutOutputDuringTests="true"
+        beStrictAboutTestsThatDoNotTestAnything="true"
+        bootstrap="./vendor/squizlabs/php_codesniffer/tests/bootstrap.php"
```

Comment:
Would be worth having custom `bootstrap.php` like https://github.com/slevomat/coding-standard/blob/master/tests/bootstrap.php.